src/number_theory/bernoulli.lean

```diff
@@ -5,6 +5,10 @@ Authors: Johan Commelin, Kevin Buzzard
 -/
 import data.rat
 import data.fintype.card
+import data.nat.factorial
+import data.nat.choose.basic
```

Comment:
```suggestion
```
You don't need either of these. Of course `data.nat.choose.basic` will import `data.nat.factorial` but somehow they're both being imported by something else anyway, possibly the power series stuff.